### PR TITLE
feat(anta): Refactor VerifyReachability test case to add coverage for DF Bit, packet size

### LINKED ISSUE #756: Refactor VerifyReachability test case to add coverage for DF Bit, packet size
We need to refactor the existing VerifyReachability test case to enhance coverage including additional scenarios. Currently the test case does not cover:

1. **DF (Don't Fragment) Bit**:  We need to test packets with the DF bit set to ensure proper handling of fragmentation.
2. **Packet sizes**: The test case should cover a range of packet size to verify behavior under different conditions.

**Tasks:**
- Update the test case to include optional field of DF bit(Update input model for df-bit) in command.
- Include optional field of packet size variation(Update input model for packet size) in command.

**Example of ping commands:**
ping $dest_ip source Loopback0 size 9000 df-bit repeat 10
ping $LO0P_IP source Loopback0 size 9100 df-bit repeat 10

### PR DESCRIPTION
**Pull request by vitthalmagadum**
# Description

- Updated the test case to include optional field of DF bit(Updated input model for df-bit) in command.
- Included optional field of packet size variation(Updated input model for packet size) in command.

Fixes #756 

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have run pre-commit for code linting and typing (`pre-commit run`)
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes (`tox -e testenv`)

### CODE REVIEW

File: anta/tests/connectivity.py [hunk 1]
```diff
@@ -61,15 +69,27 @@ class Host(BaseModel):
             """VRF context. Defaults to `default`."""
             repeat: int = 2
             """Number of ping repetition. Defaults to 2."""
+            size: int = 100
+            """Specify datagram size. Defaults to 100"""
```
Comment (carl-baillargeon) [inline anta/tests/connectivity.py:73]:
```suggestion
            """Specify datagram size. Defaults to 100."""
```

File: anta/tests/connectivity.py [hunk 2]
```diff
@@ -61,15 +69,27 @@ class Host(BaseModel):
...
+            df_bit: bool = False
+            """Enable do not fragment bit in IP header. Defaults to False"""
```
Comment (carl-baillargeon) [inline anta/tests/connectivity.py:75]:
```suggestion
            """Enable do not fragment bit in IP header. Defaults to False."""
```

File: anta/tests/connectivity.py [hunk 3]
```diff
@@ -33,16 +33,24 @@ class VerifyReachability(AntaTest):
             - source: Management0
               destination: 1.1.1.1
               vrf: MGMT
+              df_bit: True
+              size: 100
```
Comment (carl-baillargeon) [inline anta/tests/connectivity.py:37]: Can you also update `anta/examples/tests.yaml` with these add-ons?

File: anta/tests/connectivity.py (hunk 1 above)
Comment (vitthalmagadum) [reply anta/tests/connectivity.py]: Handled.

File: anta/tests/connectivity.py (hunk 2 above)
Comment (vitthalmagadum) [reply anta/tests/connectivity.py]: Handled.

File: anta/tests/connectivity.py (hunk 3 above)
Comment (vitthalmagadum) [reply anta/tests/connectivity.py:37]: Added these add-ons to` anta/examples/tests.yaml`
Thanks!

**Review by carl-baillargeon — APPROVED** [review]
LGTM

**Pull request merged**